docs/application/native/guides/complications/watchface-complication.md

```diff
@@ -128,20 +128,54 @@ To get the complication provider data, the following functions can be used:
 | `watchface_complication_data_get_long_text()` | long text |
 | `watchface_complication_data_get_title()` | title text |
 | `watchface_complication_data_get_timestamp()` | timestamp |
+| `watchface_complication_data_get_timeinfo()` | time information |
+| `watchface_complication_timeinfo_get_timezone()` | timezone |
+| `watchface_complication_timeinfo_get_timezone_id()` | timezone ID |
+| `watchface_complication_timeinfo_get_timezone_country()` | timezone country |
+| `watchface_complication_timeinfo_get_timezone_city()` | timezone city |
+| `watchface_complication_timeinfo_get_timestamp()` | timestamp |
 | `watchface_complication_data_get_image_path()` | image path |
 | `watchface_complication_data_get_ranged_value()` | min / max / current value |
 | `watchface_complication_data_get_icon_path()` | icon path |
 | `watchface_complication_data_get_extra_data()` | extra data |
 
+> **Note**

```

Comment:
Suggested rephrase for the note:

> **Note**
>
>   Use `watchface_complication_data_get_timeinfo()`, as `watchface_complication_data_get_timestamp()` is deprecated since Tizen 5.5.

Reply:
@rishu-samsung Would you please refer https://github.com/Samsung/tizen-docs/pull/856/files?file-filters%5B%5D=.md#r354057699?

Reply:
@mijin85cho watchface_complication_data_get_timestamp returns long type and watchface_complication_timeinfo_get_timestamp returns time_t.
Also, watchface_complication_timeinfo_get_timestamp is generated by framework based on timezone id.
On the other hand, watchface_complication_data_get_timestamp API just returns value that set by provider.